### DISCUSSION TITLE
Welcome to Eternal Safe's help center!

### DISCUSSION THREAD

**Discussion by devanoneth** [eternalsafe/wallet#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to answer any questions you may have on Eternal Safe. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

To get started, please find the section below that's most relevant to you.

### I encountered a bug
Please check if someone has already encountered your issue. If not, please [open a bug report](https://github.com/eternalsafe/wallet/issues/new?assignees=&labels=bug&projects=&template=bug-report.md).

### I found a security vulnerability
Please DM @devanoneth on X.

### I have a feature request
Please [open a feature request](https://github.com/eternalsafe/wallet/issues/new?assignees=&labels=enhancement&projects=&template=feature-request.md).

### I want to contribute
Please first read our [contributing guidelines](https://github.com/eternalsafe/wallet/blob/eternal-safe/CONTRIBUTING.md). Then, [open a feature request](https://github.com/eternalsafe/wallet/issues/new?assignees=&labels=enhancement&projects=&template=feature-request.md) or [take a look at some open issues](https://github.com/eternalsafe/wallet/issues?q=is%3Aopen+is%3Aissue).

**Comment by gregor4x** [comment]
Hello
i can not load safe wallet, got error
Address given is not a valid Safe account...

i was able to load it on https:// app.safe.global / new-safe / load ,so is valid

**Reply by gregor4x** [reply]
thats a problem with RPC, only few works

**Comment by 0ldSaltSailor61** [comment]
over the last few dayus I have been attempting to link my OneKey Pro Hardware Wallet with my Safe Wallet so that I could move forward with staking for my Gnosis Nodes. I have a DappNode Gnosis Chain Node which came with 3 GNOs embedded which I wanted to add at least 1 possibly 2 additional GNOs to for a total of 4 or 5 Nodes. but each time I attempted to link up my OneKey Wallet to my Safe Wallet at the point of linkage with safe the crypto would disappear.

**Comment by DRIPDROPS** [comment]
![image](https://github.com/user-attachments/assets/81520a20-eecb-49a1-9f08-8889624eb383)
im trying to use mantle network via your link url method and getting this, using this:

https://eternalsafe.eth.limo/?chainId=500&chain=Mantle&shortName=mantle%rpc=https%3A%2F%2Fmantle.drpc.org&currency=MNT&symbol=MNT&expAddr=https%3A%2F%2Fexplorer.mantle.xyz%2Faddress%2F%7B%7Baddress%7D%7D&expTx=https%3A%2F%2Fexplorer.mantle.xyz%2Ftx%2F%7B%7Bhash%7D%7D&l2=true&testnet=tfalse

**Reply by DRIPDROPS** [reply]
same result with correct chainID what are chances of a list covering all that SAFE covers and a REAL favor would be a DRAIN ACCOUNT APP present, they took it to far with the power trip cutting everyone out of their finances.

**Reply by devanoneth** [reply]
@DRIPDROPS you have typos in your URL:
 - you have a `%` instead of a `&` before `rpc=`. 
 - `testnet=tfalse` instead of `testnet=false` (but this is optional anyway)

This link works:
https://eternalsafe.eth.limo/?chainId=500&chain=Mantle&shortName=mantle&rpc=https%3A%2F%2Fmantle.drpc.org&currency=MNT&symbol=MNT&expAddr=https%3A%2F%2Fexplorer.mantle.xyz%2Faddress%2F%7B%7Baddress%7D%7D&expTx=https%3A%2F%2Fexplorer.mantle.xyz%2Ftx%2F%7B%7Bhash%7D%7D&l2=true&testnet=false

**Reply by DRIPDROPS** [reply]
awesome that it loads but nmo connecty i will troubleshoot further as well

**Comment by 0ldSaltSailor61** [comment]
I was told that the crypto
was already set aside in a wallet waiting for me. Wvwn I knew that was bullshit! I mean it be nice bur I knew it was bullshit
